codegen.ts

```diff
@@ -4,7 +4,7 @@ const config: CodegenConfig = {
   overwrite: true,
   schema: [
     {
-      ['https://management.api.basta.wtf/graphql']: {
+      ['https://management.api.basta.ai/graphql']: {
```

Comment:
@EinarLogi since we're dev-ing against `staging`, I think need this to be `.wtf`. Or else we would need to deploy all our backend to `production` before being able to run `codgen`.

Reply:
Can you run codegen against .wtf and push the changes? Thanks!